.github/workflows/continous-delivery.yml

```diff
@@ -127,6 +127,7 @@ jobs:
           --container-mount-disk mount-path="/zebrad-cache",name=zebrad-cache-${{ env.GITHUB_SHA_SHORT }} \
           --machine-type ${{ env.MACHINE_TYPE }} \
           --scopes cloud-platform \
+          --labels=app=zebrad,environment=prod,github_ref=${{ env.GITHUB_REF_SLUG_URL }} \
```

Comment:
This is unrelated to this change, but there is an `inputs.network`, `env.NETWORK`, and hard-coded `Mainnet` in this file.

If we want to add the network here, we could make them all use `inputs.network`?

Reply:
Fixed by [chore: add Network as a label](https://github.com/ZcashFoundation/zebra/pull/5693/commits/146f747d501bec56380aeb630b5398eb9c94df37)

Standardizing the networks, which is a bigger job, might be better doing it in a separate PR (as this is something we should do)

Reply:
The labels will be incorrect without the network fix, what do you think of #5710?

I'm happy to merge that PR separately if you want.